CONTRIBUTING.md

```diff
@@ -31,16 +31,18 @@ $ ./gradlew clean dokka :arrow-docs:runAnk
 
 The artifact will reside in `modules/docs/arrow-docs/build/site`. The artifact ΛNK produces could be served as a website by [jekyll](https://jekyllrb.com/), a static site generator.
 
-Jekyll is written in ruby, so you need ruby [gem](https://rubygems.org/pages/download) to install it.
+In order to setup a local version of the documentation in your machine, you will have to install [gem](https://rubygems.org/pages/download) as Jekyll is written in ruby. Afterwards, install [bundler](https://bundler.io/) to manage the project dependencies.
 
 ```
-$ gem install jekyll
+$ gem install bundler
+$ bundle install --gemfile=modules/docs/arrow-docs/Gemfile
 ```
 
-Now you have jekyll, then you can preview the documentation by serving the artifact produced by ΛNK.
+After all the dependencies are installed, you can preview the documentation by serving the artifact produced by ΛNK.
 
 ```
-$ jekyll serve --source modules/docs/arrow-docs/build/site/
+$ cd modules/docs/arrow-docs/
```

Comment:
☝️ Have not been able to make this work without switching folder first :(

Reply:
🤔 weird no? Will give it a try

Reply:
Notice I had to install bundler so it manages the dependencies

Reply:
I think what's happening is that considering you are installing the gem dependencies on that docs directory, when you run `bundle exec` you are calling the installed dependency, so you need to be in the directory where it is installed.

I propose to change the order to be:

```sh
$ gem install bundler

$ cd modules/docs/arrow-docs/
$ bundle install --path vendor/bundle
$ bundle exec jekyll serve --source build/site
```

This will install the dependencies locally, which I also feel safer, so we will be needing to update the `.gitignore` file to something like:

```
build

## Jekyll
_site
.sass-cache
.jekyll-metadata

## OSX shit
.DS_Store

## Ruby environment normalization:
/.bundle/
/vendor/bundle
/lib/bundler/man/
```